app/__init__.py

```diff
@@ -20,12 +30,14 @@ def create_app(config_name):
 
     if application.config['VCAP_SERVICES']:
         cf_services = json.loads(application.config['VCAP_SERVICES'])
+        service = get_service_by_name_from_vcap_services(cf_services, 'search_api_elasticsearch')
```

Comment:
Think we should consider making this an app config variable rather than hard coding but happy to do that as a different pr

Reply:
That's a nice idea. What would we call it, DM_ELASTICSEARCH_SERVICE_NAME?

Reply:
Sounds very reasonable :)